config-object-store/src/main/java/org/hypertrace/config/objectstore/ConfigsResponse.java

```diff
@@ -0,0 +1,9 @@
+package org.hypertrace.config.objectstore;
+
+import java.util.List;
+
+public interface ConfigsResponse<T> {
+  List<ContextualConfigObject<T>> getContextualConfigObjects();
```

Comment:
Do we want to use `List<ContextualConfigObject<T>>` or `List<T>` ? Even though the latter is derivable from `ContextualConfigObject`, the vast majority of our use cases are after `T` directly so by returning this we're introducing an extra mapping for every consumer. Suggest return `List<T>` and we can always add the more complete/verbose form if a use case comes along.

Reply:
Updated